spec/jobs/scheduled_publishing_job_spec.rb

```diff
@@ -12,7 +12,7 @@
       it "calls PublishService if edition can be published" do
         edition = create(:edition,
                          :scheduled,
-                         scheduled_publishing_datetime: Time.current)
+                         scheduling: build(:scheduling, publish_time: Time.current))
```

Comment:
Just double-checking we still want to manually build a scheduling here?

Reply:
Oops missed.